### DISCUSSION TITLE
Writing changes back to SVG

### DISCUSSION THREAD

**Discussion by Shujee** [ElinamLLC/SharpVectors#248] [Q&A]
Does SharpVector support converting DrawingGeometry back to SVG file? I'm trying to create a simple SVG editor in WPF, where users will be able to open SVG files, make minor changes (add a circle, rect or path, delete one or more elements, change fill/stroke etc.) and then save those changes back to the SVG file. Is SharpVectors good enough for performing round-trip conversion between WPF objects and SVG elements?

I'm also interested in keeping this as much MVVM-compatible as possible (e.g. keeping most of the logic out of the UI layer by, for example, using data binding with SharpVectors WPF controls instead of calling their methods).

**Comment by paulushub** [comment]
> Does SharpVector support converting DrawingGeometry back to SVG file?

No, it is currently one way.
